EnglishTranslation-master/SKILL.tsv

```diff
@@ -8620,9 +8620,9 @@ SKILL_20170420_008619	Attack #{SkillFactor}#% x no. of hits{nl}Arrow Rain Durati
 SKILL_20170420_008622	Attack: #{SkillFactor}#%{nl}Recovers HP equal to 5% of your maximum HP{nl}Additional HP Recovery Rate: #{CaptionRatio}# ~ #{CaptionRatio2}#{nl}Magic Circle Duration: #{CaptionTime}# seconds	공격력 #{SkillFactor}#%{nl}최대 HP의 5% 회복{nl}추가 HP 회복량 #{CaptionRatio}# ~ #{CaptionRatio2}#{nl}마법진 지속시간 #{CaptionTime}#초	SKILL_20160718_006380	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{KDownPower}#%{nl}Recovers HP equal to 5% of your maximum HP{nl}Additional HP Recovery Rate: #{CaptionRatio}# ~ #{CaptionRatio2}#{nl}Magic Circle Duration: #{CaptionTime}# seconds	공격력 #{SkillAtkAdd}#{nl}특성 공격력 #{KDownPower}#%{nl}최대 HP의 5% 회복{nl}추가 HP 회복량 #{CaptionRatio}# ~ #{CaptionRatio2}#{nl}마법진 지속시간 #{CaptionTime}#초	
-SKILL_20170420_008623	#{CaptionRatio3}# second attack #{SkillFactor}#%{nl}No. of hits: #{CaptionRatio}#{nl}Magic circle duration #{CaptionTime}#seconds	#{CaptionRatio3}#초당 공격력 #{SkillFactor}#%{nl}공격 횟수 #{CaptionRatio}#회{nl}마법진 지속시간 #{CaptionTime}#초	SKILL_20150714_004210	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Number of Hits: #{CaptionRatio}#{nl}Magic Circle Duration: #{CaptionTime}# seconds	공격력 #{SkillAtkAdd}#{nl}특성 공격력 #{CaptionRatio2}#%{nl}공격 횟수 #{CaptionRatio}#회{nl}마법진 지속시간 #{CaptionTime}#초	
-SKILL_20170420_008624	Decreases 0.3% per stack{nl}Magic circle duration #{CaptionTime}# seconds	중첩 당 0.3%씩 감소{nl}마법진 지속시간 #{CaptionTime}#초	SKILL_20150317_001436	Defense: -#{CaptionRatio}#{nl}Per Stack: -#{CaptionRatio2}#{nl}Magic Circle Duration: #{CaptionTime}# seconds	방어력 #{CaptionRatio}# 감소{nl}중첩 당 #{CaptionRatio2}#씩 감소{nl}마법진 지속시간 #{CaptionTime}#초	
-SKILL_20170420_008625	0.3 second attack #{SkillFactor}#%{nl}No. of hits #{CaptionRatio}#{nl}Magic circle duration 30 seconds	0.3초당 공격력 #{SkillFactor}#%{nl}공격 횟수 #{CaptionRatio}#회{nl}마법진 지속시간 30초	SKILL_20150714_004211	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Number of Hits: #{CaptionRatio}#{nl}Magic Circle Duration: 30 seconds	공격력 #{SkillAtkAdd}#{nl}특성 공격력 #{CaptionRatio2}#%{nl}공격 횟수 #{CaptionRatio}#회{nl}마법진 지속시간 30초	
+SKILL_20170420_008623	#{CaptionRatio3}# second attack #{SkillFactor}#%{nl}No. of hits: #{CaptionRatio}#{nl}Magic Circle Duration #{CaptionTime}#seconds	#{CaptionRatio3}#초당 공격력 #{SkillFactor}#%{nl}공격 횟수 #{CaptionRatio}#회{nl}마법진 지속시간 #{CaptionTime}#초	SKILL_20150714_004210	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Number of Hits: #{CaptionRatio}#{nl}Magic Circle Duration: #{CaptionTime}# seconds	공격력 #{SkillAtkAdd}#{nl}특성 공격력 #{CaptionRatio2}#%{nl}공격 횟수 #{CaptionRatio}#회{nl}마법진 지속시간 #{CaptionTime}#초	
```

Comment:
Colon after `Magic Circle Duration`
`#{CaptionTime}#seconds` -> `#{CaptionTime}# seconds` (space between '#' and 'seconds')